app/controllers/mainsearch.php

```diff
@@ -54,6 +54,17 @@
     require_once MODELS . '3locales_search.php';
 }
 
+/*
+    Trim the search query when searching for entities, display a warning.
+    This needs to be done before including search_form.php, in order to
+    display the trimmed value in the search field.
+*/
+if ($search->getSearchType() == 'entities' &&
+    $search->getSearchTerms() !== trim($search->getSearchTerms())) {
+    $search->setSearchTerms(trim($search->getSearchTerms()));
```

Comment:
I guess Atom's autocomplete here failed me, and it was using `setRegexSearchTerms` before.